corehq/apps/case_importer/util.py

```diff
@@ -184,12 +184,9 @@ def get_spreadsheet(filename):
 
 
 def get_importer_error_message(e):
-    if isinstance(e, ImporterRefError):
-        # I'm not totally sure this is the right error, but it's what was being
-        # used before. (I think people were just calling _spreadsheet_expired
-        # or otherwise blaming expired sessions whenever anything unexpected
-        # happened though...)
-        return _('Sorry, your session has expired. Please start over and try again.')
+    if isinstance(e, ImporterCacheError):
+        return _('The file you uploaded cannot be read successfully. '
```

Comment:
I am a bit concerned this implies there is something wrong with the file, which I think turned out not to be the case. I agree with the next line about trying again, but would love to change this to make it clear the error is on our end (the file is missing), not that the file itself needs to be changed (cannot be read as is).